components/SubmissionForm.vue

```diff
@@ -232,29 +295,38 @@ const initialValidationCheck = async (inputValue: string, field: string) => {
     }
 }
 
-watch(() => submissionStore.location, newValue => {
+watch(() => location.value, newValue => {
     if (validationCheckedPreviously.googleMapsUrl.value) {
         isValidInput.googleMapsUrl.value = validations.validateGoogleMapsUrlInput(newValue)
     }
 })
-watch(() => submissionStore.lastName, newValue => {
+watch(() => lastName.value, newValue => {
     if (validationCheckedPreviously.lastName.value) {
         isValidInput.lastName.value = validations.validateUserSubmittedLastName(newValue)
     }
 })
-watch(() => submissionStore.firstName, newValue => {
+watch(() => firstName.value, newValue => {
     if (validationCheckedPreviously.firstName.value) {
         isValidInput.firstName.value = validations.validateUserSubmittedFirstName(newValue)
     }
 })
-watch(() => submissionStore.selectLanguage1, newValue => {
+watch(() => selectLanguage1.value, newValue => {
     if (validationCheckedPreviously.primarySpokenLangauge.value) {
         isValidInput.primarySpokenLangauge.value = validations.validateFirstSpokenLanguage(newValue)
     }
 })
-watch(() => submissionStore.selectLanguage2, newValue => {
+watch(() => selectLanguage2.value, newValue => {
     if (validationCheckedPreviously.secondarySpokenLanguage.value) {
         isValidInput.secondarySpokenLanguage.value = validations.validateSecondSpokenLanguage(newValue)
     }
 })
+
+onMounted(async () => {
+    toast = useToast()
+
+    resetForm()
+    await nextTick()
+
+    isSubmissionFormReady.value = true
```

Comment:
nit: could make this slightly more clear to indicate it's around being loaded or not like `isFormLoaded` or `isFormInitialized`